retrofit/src/main/java/retrofit2/Retrofit.java

```diff
@@ -143,6 +144,8 @@
             }
             if (platform.isDefaultMethod(method)) {
               return platform.invokeDefaultMethod(method, service, proxy, args);
+            } else if (Modifier.isStatic(method.getModifiers())) {
```

Comment:
Let me know if you want this removed. It's weird to be calling a static method on an instance, but technically legal. This at least keeps that behavior

Reply:
Is this line reachable in test cases? I assume it won’t be dispatched this way

Reply:
Yeah I don't think this can happen. The only case I can think of is if a virtual method was made static without recompiling the caller. But even then I would expect the VM to intervene.

Reply:
done f1fbbda

Reply:
Is the condition reachable?

Reply:
Ahh, this happens during reflection, not invocation. Works for me.